src/Analyzer.Cli.FunctionalTests/CommandLineParserTests.cs

```diff
@@ -334,9 +334,8 @@ public void FilterRules_ValidConfig_RulesFiltered(bool isBicep, string configNam
 
                 var cliConsoleOutput = outputWriter.ToString();
 
-                // All JSON rules are filtered out; PSRule rules are currently not filtered by the config file and should appear in the output
                 Assert.IsTrue(!cliConsoleOutput.Contains("TA-"));
-                Assert.AreEqual((int)ExitCode.Violation, result.Result);
+                Assert.AreEqual((int)ExitCode.Success, result.Result);
```

Comment:
Did you confirm the rule you removed was previously the only failure here?

Reply:
Yes.

Reply:
For the record, I left the PSRule comment there for the future. If a future PSRule shows up, it would help future dev here with additional context and save investigation time.

Reply:
Not sure what you mean - the only comments I see related to PSRule was removed in your changes.